src/components/InvoiceItem.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import { TextField, Grid } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+export default function InvoiceItem() {
+  return (
+    <>
+      {" "}
+      <Grid item xs={6} style={{ paddingLeft: 30 }}>
```

Comment:
nie uzywaj propa style, jest on odpowiednikiem stylowania inlineowego. Zapoznaj się z propami z material ui, sx jest sposobem na stylowanie wewnątrz komponentów, ale daje więcej możliwosci (np wyciąganie danych z theme, czy rozne wartosci dla danego property w zaleznosci do breakpointu). Do ustawiania paddingów czy marginesów masz też osobne propy: `px` nadaje padding w osi x, `py` analogicznie, `pb` to padding bottom, masz tez `pl`, `pt`, `pr` i analogicznie dla marginesu `mx, my, mb, ml, mt, mr`. 

https://mui.com/system/getting-started/the-sx-prop/
https://mui.com/system/spacing/